addons/spreadsheet/static/src/o_spreadsheet/o_spreadsheet.js

```diff
@@ -18094,7 +18093,7 @@ function getChartMenuActions(figureId, onFigureDeleted, env) {
                     "text/html": innerHTML,
                     "image/png": blob,
                 });
-                env.notifyUser({ sticky: false, type: "success", text: _t("Chart copied to clipboard") });
+                env.notifyUser({ type: "success", text: _t("Chart copied to clipboard") });
```

Comment:
`o_spreadsheet.js` cannot be modified, it should be updated by making a PR in https://github.com/odoo/o-spreadsheet.
In the meantime, you could update `notifyUser` of `addons/spreadsheet/static/src/hooks.js` with the following:

```js
function notifyUser(notification) {
    notifications.add(notification.text, {
        type: notification.type,
        autocloseDelay: notification.sticky ? 0 : undefined,
    });
}
```
It should work without updating o-spreadsheet :)